Client/Frontend/Browser/BrowserViewController/BrowserViewController+ProductNotification.swift

```diff
@@ -10,6 +10,45 @@ import Shared
 // MARK: - ProductNotification
 
 extension BrowserViewController {
+
+    // MARK: BenchmarkTrackerCountTier
+    
+    enum BenchmarkTrackerCountTier: Int, Equatable, CaseIterable {
+        case specialTier = 1000
+        case newbieExclusive = 5000
+        case casualExclusive = 10000
+        case regularExclusive = 25000
+        case expertExclusive = 75000
+        case professionalTier = 100000
+        case primeTier = 250000
+        case grandTier = 500000
+        case legendaryTier = 1000000
+
+        var title: String {
+            switch self {
+                case .specialTier:
+                    return Strings.ShieldEducation.benchmarkSpecialTierTitle
+                case .newbieExclusive, .casualExclusive, .regularExclusive, .expertExclusive:
+                    return Strings.ShieldEducation.benchmarkExclusiveTierTitle
+                case .professionalTier:
+                    return Strings.ShieldEducation.benchmarkProfessionalTierTitle
+                case .primeTier:
+                    return Strings.ShieldEducation.benchmarkPrimeTierTitle
+                case .grandTier:
+                    return Strings.ShieldEducation.benchmarkGrandTierTitle
+                case .legendaryTier:
+                    return Strings.ShieldEducation.benchmarkLegendaryTierTitle
+            }
+        }
+
+        var nextTier: BenchmarkTrackerCountTier? {
```

Comment:
this could get a unit test, to check next value and last value(all tiers passed)

Reply:
I will add test for nextTier functionalty